ykman-gui/qml/TouchYubiKey.qml

```diff
@@ -0,0 +1,16 @@
+import QtQuick 2.5
+import QtQuick.Dialogs 1.2
+
+MessageDialog {
+    icon: StandardIcon.Information
+    title: qsTr("Touch your YubiKey")
+    text: qsTr("Please touch your YubiKey...")
+    standardButtons: StandardButton.NoButton
+
+    function show(message) {
```

Comment:
Is this needed? Why not just change the text property directly if needed?

Reply:
It's not strictly necessary, no, but I prefer encapsulating the details in a method instead of having callers poke directly at the internals. It also makes more sense at the call site to me to do both parts of the one thing (display a message) in one statement.

Reply:
I'd suggest we  add override-text-functionality when actually needed then.

Reply:
Ah, right - I _did_ call this as something like `touchYubiKey.show("Please touch your YubiKey to unlock the management key...")` from PIV manager at one point, that's why. But that is no longer the case.